examples/graphbolt/rgcn/hetero_rgcn.py

```diff
@@ -663,7 +670,15 @@ def main(args):
         "--dataset",
         type=str,
         default="ogbn-mag",
-        choices=["ogbn-mag", "ogb-lsc-mag240m"],
+        choices=[
+            "ogbn-mag",
+            "ogb-lsc-mag240m",
+            "igb-heterogeneous-tiny",
+            "igb-heterogeneous-small",
+            "igb-heterogeneous-medium",
+            "igb-heterogeneous-large",
+            "igb-heterogeneous-full",
```

Comment:
Do you think it is better to follow the convention in the [paper](https://dl.acm.org/doi/pdf/10.1145/3580305.3599843)?
```suggestion
            "igb-het-tiny",
            "igb-het-small",
            "igb-het-medium",
            "igb-het-large",
            "igb-het",
```